wireless_modules/wireless_module.py

```diff
@@ -38,6 +42,22 @@ def __init__(self, module_id, battery_reader=None):
         self.mqtt = Client(client_id, config.MQTT_BROKER)
 
         self.battery = battery_reader
+        self.status_led = status_led
+
+        asyncio.get_event_loop().set_exception_handler(self.error_handler)
+
+    def error_handler(self, _loop, context):
+        self.status_led.set_state(WmState.Error)
+        print("An error occured in a uasyncio task!")
+        print(context["message"])
+        sys.print_exception(context["exception"])
```

Comment:
Do we need both these print statements?

Reply:
Yep, so by memory the first one will print whatever message is associated with the exception (e.g. `Task exception wasn't retrieved`, looking at the example above) and then the next line prints the actual traceback.

Reply:
I thought tracebacks themselves print out the actual message as well